README.md

```diff
@@ -168,7 +165,7 @@ Asset animation always requires the `bevy_asset` feature.
 
```

Comment:
I can't comment above this but in L155 `Style` should be replaced with `Node` and in L157 `Text` with `TextColor` probably?